torch/utils/cpp_extension.py

```diff
@@ -325,6 +325,18 @@ def get_default_build_root() -> str:
     """
     return os.path.realpath(torch._appdirs.user_cache_dir(appname='torch_extensions'))
 
+def remove_default_build_root():
```

Comment:
I'm not sure we really want a public API that rm -rf a folder like this.
It makes sense as a utility for our testing framework. But I don't think we should do it here.